kiwi_lint/db_columns.py

```diff
@@ -0,0 +1,20 @@
+from pylint import interfaces, checkers
+from pylint.checkers import utils
+
+
+class AutoFieldChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'db-columns-checker'
+
+    msgs = {
+        'E4480': (
```

Comment:
4811, order is from top to bottom in the way checkers are registered in `__init__.py`

Reply:
^^^